vdom/core.py

```diff
@@ -219,7 +228,17 @@ def from_dict(cls, value):
             key=value.get('key')
         )
 
+upper = re.compile(r'[A-Z]')
+def _upper_replace(matchobj):
+    return '-' + matchobj.group(0).lower()
 
+def convert_style_names(style):
+    """Converts style names from DOM to css styles.
+    """
+    for k, v in style:
+        yield re.sub(upper, _upper_replace, k), v
```

Comment:
Could you write tests for at least just the style name conversions? That'll make me much more comfortable with this in general. 😄 

This might be better as `convert_style_key` since the dict returned here ends up getting iterated over anyways and value doesn't even get used in this function.

Once that's done, it would be great to have a docstring with examples:

```python
>>> convert_style_key("backgroundColor")
"background-color"
```

Reply:
These are great suggestions! they make the code to use it cleaner too!